remo/featuredrep/models.py

```diff
@@ -2,7 +2,6 @@
 from django.db import models
 from django.utils.timezone import now
 
-
```

Comment:
this leads to a failing flake8 lint: https://travis-ci.org/mozilla/remo/jobs/360278965